app/pages/home-for-user/news-pullout.jsx

```diff
@@ -0,0 +1,133 @@
+      projects: [],
+    };
+  },
+
+  componentDidMount() {
+    this.fetchNewestProject();
+    this.recentPublications();
+  },
+
+  recentPublications() {
+    const articles = [];
+    Object.keys(Publications).forEach((category) => {
+      Publications[category].map((project) => {
+        return articles.push(...project.publications);
+      });
+    });
+    const newestPublications = articles.sort((a, b) => {
+      return new Date(b.date) - new Date(a.date);
+    }).splice(0, 3);
+    return this.setState({
+      publications: newestPublications,
+    });
+  },
+
+  fetchNewestProject() {
+    apiClient.type('projects').get({
+      sort: '-launch_date',
+      launch_approved: true,
+      page_size: 1,
+      include: ['avatar'],
+    })
+    .then(([project]) => {
+      return new Promise(() => {
+        return apiClient.type('avatars')
+        .get(project.links.avatar.id)
+        .catch(() => {
+          return null;
+        })
+        .then((avatar) => {
+          this.setState({
+            newestAvatar: avatar,
+            newestProject: project,
+          });
+        });
+      });
+    });
+  },
+
+  renderPublication(article) {
+    return (<div key={Math.random()}>
+      <h5 className="home-page-news-pullout news-section__timestamp">{article.date}</h5>
+      <p className="home-page-news-pullout news-section__content">
+        <a href={article.href} target="_blank">
+          {article.citation}
+          <i className="fa fa-long-arrow-right"></i>
+        </a>
+      </p>
+    </div>);
+  },
+
+  renderUpdatedProjects(project) {
+    const link = window.location.origin + '/projects/' + project.slug;
+    const timestamp = moment(new Date(project.updated_at)).fromNow();
+    return (<div key={project.id}>
+      <a href={link} target="_blank">
+        <h5 className="home-page-news-pullout news-section__title">{project.name} </h5>
+        <h5> has been updated! </h5>
+        <p className="home-page-news-pullout news-section__timestamp">{timestamp}</p>
+      </a>
+    </div>);
+  },
+
+  render() {
+    const projLink = window.location.origin + '/projects/' + this.state.newestProject.slug || null
+    const avatarSrc = !!this.state.newestAvatar ? this.state.newestAvatar.src : null;
+
+    return (
+      <div className={"home-page-news-pullout news-main" + (this.props.showNews ? " active" : "")}>
+        <div className="home-page-news-pullout news-container">
+          <h2> Zooniverse News </h2>
+
+          <div className="home-page-news-pullout news-section">
+            <h4> Recent Publications </h4>
+            {this.state.publications.map((article) => {
+              return this.renderPublication(article);
+            })}
+          </div>
+
+          <div className="home-page-news-pullout news-section">
+            <h4> Newest Project </h4>
+            <ProjectCard href={projLink} key={this.state.newestProject.id} project={this.state.newestProject} imageSrc={avatarSrc} />
```

Comment:
Yes, this is the issue with the pullout. I can perform a similar solution to this as I did by including `browserHistory`, but I'm hesitant to provide all of that code across several components to check if another component is outside of the child's browser history (eg. `NewsPullout` to `ProjectIcon` to `FlexibleLink`.